spec/forms/two_factor_options_form_spec.rb

```diff
@@ -62,5 +62,34 @@
         subject.submit(selection: 'auth_app')
       end
     end
+
+    context 'when phone is selected as their first authentication method' do
+      before do
+        allow(IdentityConfig.store).to receive(:select_multiple_mfa_options).and_return(true)
+      end
+
+      it 'does not submit the phone when selected as the first single option' do
+        %w[phone].each do |selection|
+          result = subject.submit(selection: selection)
+
+          expect(result.success?).to eq false
+        end
+      end
+    end
+
+    context 'when a user wants to select phone as their second authentication method' do
+      let(:user) { create(:user, :with_authentication_app) }
+      before do
+        allow(IdentityConfig.store).to receive(:select_multiple_mfa_options).and_return(true)
+      end
+
+      it 'submits the form' do
+        %w[phone].each do |selection|
+          result = subject.submit(selection: ['phone'])
+
+          expect(result.success?).to eq true
+        end
```

Comment:
The loops here don't seem to be necessary for a single item, and in the second case we're already not using the loop parameter.

```suggestion
        result = subject.submit(selection: ['phone'])

        expect(result.success?).to eq false
      end
    end

    context 'when a user wants to select phone as their second authentication method' do
      let(:user) { create(:user, :with_authentication_app) }
      before do
        allow(IdentityConfig.store).to receive(:select_multiple_mfa_options).and_return(true)
      end

      it 'submits the form' do
        result = subject.submit(selection: ['phone'])

        expect(result.success?).to eq true
```